### PR TITLE
style(suite): implement colors from Figma based on Palette

### LINKED ISSUE #10721: add Elevations for buttons
add colors for every button variant on every elevation

(David needs to provide colors first)

related issue: https://github.com/trezor/trezor-suite/issues/10689

### LINKED ISSUE #10960: Empower/ Buy button is not visible
Assets - grid view - buy button invisible

image

### PR DESCRIPTION
**Pull request by peter-sanderson**
## Description

Adds a palette from export from Figma. This shall be now easy way how to synchronize colors and its tokens from Figma to the code.

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/10721 https://github.com/trezor/trezor-suite/issues/10689 https://github.com/trezor/trezor-suite/issues/10960

## Screenshots:
Normal :point_down: 

image

Hover :point_down: 
image

### CODE REVIEW

File: packages/components/src/components/buttons/buttonStyleUtils.ts [hunk 1]
```diff
@@ -107,13 +108,14 @@ export const getVariantStyle = (variant: ButtonVariant, elevation: Elevation) =>
             `;
         case 'tertiary':
             return css`
-                background: ${({ theme }) => theme[mapElevationToBackgroundTertiary[elevation]]};
+                background: ${({ theme }) =>
+                    theme[mapElevationToBackgroundTertiaryNormal[elevation]]};
```
Comment (jvaclavik) [inline packages/components/src/components/buttons/buttonStyleUtils.ts:118]: I'm thinking about using some functions with easier names for these extra-long maps. WDYT?

**Review by jvaclavik — APPROVED** [review]
I like it :) just a small comment

File: packages/components/src/components/buttons/buttonStyleUtils.ts (hunk 1 above)
Comment (peter-sanderson) [reply packages/components/src/components/buttons/buttonStyleUtils.ts:118]: Good idea, I have the same plans, but its probably for another day.

Comment (jvaclavik) [reply packages/components/src/components/buttons/buttonStyleUtils.ts:118]: OK, just please create an issue for it in github 🙏

**Review by dahaca — APPROVED** [review]
🔥🔥🔥

**Pull request merged**